### PR TITLE
Create TOS.html

### PR DESCRIPTION
**Pull request by CastyiGlitchxz**
I create a html file for the terms of service and rules.

### CODE REVIEW

**Review by 01000011-00110111 — CHANGES_REQUESTED** [review]
Add some styling and add some example text for us too se how everything works

File: TOS.html [hunk 1]
```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8">
```
Comment (cserver45) [inline TOS.html:5]: Need to add the other <meta> tags we have in index.html

File: TOS.html [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+  <meta name="viewport" content="width=device-width">
+  <title>School Chat - Rules & TOS</title>
+  <link href="style.css" rel="stylesheet" type="text/css" />
```
Comment (cserver45) [inline TOS.html:8]: Should we just create a new style.css file just for this? or just try to use as much as we can from the main style.css file?

File: TOS.html [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+          <h2>5</h2>
+        </div>
+        <button class="ToSBtn">Age Requirements</button>
+      </div>
+  
+      <div class="btndiv">
+        <div class="NumberCount">
+          <h2>6</h2>
+        </div>
+        <button class="ToSBtn">Copyright Policy</button>
```
Comment (cserver45) [inline TOS.html:53]: change this to Licence info (because this is licenced under the AGPL)

**Review by cserver45 — CHANGES_REQUESTED** [review]
I like it, but we should add in a few things

**Review by CastyiGlitchxz — COMMENTED** [review]
Nice Ideas.

**Pull request closed without merging**